app/documentation/docs/[version]/[slug]/AccordionListWrapper.tsx

```diff
@@ -20,7 +20,8 @@ export default function AccordionListWrapper({
   ) => {
     return (
       <ul className={styles.accordionMenu}>
-        {items?.map((item, index) => (
+        {items?.filter(item => parseInt(item.level) <= 3)
```

Comment:
Lets make this 2 instead right away?

Reply:
Or maybe have it as some `navigationDepth` prop or similar and pass it along from page.tsx?